src/main/kotlin/io/github/moulberry/notenoughupdates/commands/dev/NEUStatsCommand.kt

```diff
@@ -54,10 +54,13 @@ import javax.management.ObjectName
 class NEUStatsCommand {
     @SubscribeEvent
     fun onCommands(event: RegisterBrigadierCommandEvent) {
-        event.command("stats", "neustats") {
+        event.command("neustats") {
             thenLiteralExecute("modlist") {
                 clipboardAndSendMessage(
                     DiscordMarkdownBuilder()
+                        .also {
```

Comment:
Why tho? Like either we just ask for the full thing, if we need the stats, or we dont

Reply:
thats not really a bug in either case

Reply:
1. it's the old behaviour 2. the more stats the better

Reply:
1. i dont care about the old behaviour.
2. i dont really see the reason for adding this. this is the modlist command, lets just keep the modlist the modlist. removing unrelated stats from that was intentional.